app/views/users/show.html.haml

```diff
@@ -0,0 +1,13 @@
+%h1= 'Welcome text'
+= @user.email
+.row
+  %aside.col-md-4
+    %section.user_info
+      %h1
+        = user_gravatar @user
+        = @user.first_name
+= simple_form_for @user do |f|
+  = f.input :first_name, input_html: { value: 'name' }
```

Comment:
mmm, dont' think value it is good idea, maybe placeholder it is better for this purpose?